docs/modules/omevv_firmware_repository_profile.rst

```diff
@@ -0,0 +1,213 @@
+.. _omevv_firmware_repository_profile_module:
+
+
+omevv_firmware_repository_profile -- Create, modify, and delete OMEVV firmware repository profile
```

Comment:
Create, modify, or delete OMEVV firmware repository profile

(Please change in the title also)

Reply:
done